src/NuGet.Services.Metadata.Catalog.Monitoring/Model/PackageRegistrationLeafMetadata.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Newtonsoft.Json;
+using NuGet.Protocol;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    /// <summary>
+    /// The metadata for a particular package in its registration leaf.
+    /// </summary>
+    public class PackageRegistrationLeafMetadata
+    {
+        public PackageRegistrationLeafMetadata()
+        {
+        }
+
+        public PackageRegistrationLeafMetadata(V2FeedPackageInfo package)
+        {
+            Listed = package.IsListed;
+            PackageContent = package.DownloadUrl;
+            Published = package.Published;
+        }
```

Comment:
Why not keep naming consistent across classes, i.e.  DownloadUrl ; DownloadUrlEqualityComparer?